android/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,11 +1,11 @@
-• New OpenStreetMap data as of February 27
-• Wiki articles are now clickable and visible for bookmarks
-• Fixed several crashes
-• Fixed missing Traditional Chinese localization
-• Improved Help and selected POI dialogs
-• Fixed some not displayed complex opening hours
-• Fixed and improved search
-• Added healthcare specialists
-• Added communication towers, solar, wind, gas, and hydropower generators
-• Improved ar, nb, pt, and tr translations
+• New OSM Planet data as of March 26
+• Experimental Lane assistant
+• Fix wrong location arrow position during navigation
+• Fix bookmark's color chooser for Right-To-Left languages
+• Properly display speed and distance in navigation mode
+• Fix API to select a point on the map
+• Many new shop types, including cannabis one :)
```

Comment:
Move to the third position and say it explicitly, "Add cannabis shops for your convenience"

Reply:
This line also moved. The comment was about "Many new shop types, including cannabis one :)".